<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OnAir">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>반려동물 인식</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DogInfo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>사료 배급량 정보</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>반려동물 관리 시스템</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Record">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="text">
     <string>녹음</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
